dbms/tests/queries/0_stateless/00950_table_function_s3_wip/clickhouse-test

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python3
```

Comment:
It should be an integration test, see `tests/integration`.

Reply:
+